pkg/didcomm/common/service/messenger.go

```diff
@@ -6,6 +6,9 @@ SPDX-License-Identifier: Apache-2.0
 
 package service
 
+// DIDCommContextEnvelopeMediaTypeKey is DIDCommContext property key holding the DIDComm envelope's media type.
+const DIDCommContextEnvelopeMediaTypeKey = "DIDCommContextEnvelopeMediaType"
```

Comment:
will MediaType represent the envelope `typ`only or a combination of envelope `typ` + payload `cty`?

Reply:
@Baha-sk as per the RFCs, the **envelope's** media type determines both the `typ` and the `cty` values.